js/components/portal-dashboard/activity-level-feedback-student-rows.tsx

```diff
@@ -44,7 +46,7 @@ export const ActivityLevelFeedbackStudentRows: React.FC<IProps> = (props) => {
     }
 
     return (
-      <div key={index} className={css.feedbackRows__row}>
+      <div key={activityId + index.toString()} className={css.feedbackRows__row}>
```

Comment:
As long as the `activityId` is unique, you shouldn't add the `index`. In the case of re-rendering with the order changed or with elements added/removed, the index will change which will cause unnecessary element re-rendering. Using the `activityId` as the key means that the element will re-render iff it is no longer associated with the same activity, which should be what you want.

Reply:
@kswenson Thanks. I may be misunderstanding something here, but I think in this case where we have one activity and a row for each student working on that activity, `activityId` won't be unique enough. Each student row should have its own unique key value, right? If we only use `activityId`, wouldn't that key value end up being the same for every row?

Reply:
Ah, the importance of understanding the context rather than just looking at a line of code in isolation. In that case, yes, you do need more that the `activityId` or `currentQuestionId`. An ID associated with the student would be better than an index, however, if one is available, for the reasons mentioned above.

Reply:
Good point about the student ID option. I'll use that instead. Thanks!